packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/cutFromEnd..st

```diff
@@ -0,0 +1,11 @@
+attributes
+cutFromEnd: aLength
+
+	| cutDelimiter |
+	cutDelimiter := self copy
+							content: (self content last: aLength);
+							position: self endPosition - aLength + 1;
+							yourself.
+	self content: (self content copyFrom: 1 to:  self endPosition - aLength).
```

Comment:
This line needs to be changed. Using `self endPosition` doesn't make sense. We want a copy of the string except for the last `aLength` characters but `endPosition` is the position of the end of the token in the whole string that is being parsed. Instead it would make sense to use `self content first: self length - aLength`. This gives us the first `self length - aLength` characters of the content.